### PR TITLE
DEVOPS-264: setup-reserve-ip-in-VPC-module

### PR DESCRIPTION
**Pull request by aabidh-op**
DEVOPS-264: setup-reserve-ip-in-VPC-module

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -52,3 +52,9 @@ variable "private_cidr_3" {
   type        = string
   description = "Private subnet cird"
 }
+
+# variable "reserve_nat_eip" {
```
Comment (mfmsuhail) [inline variables.tf:56]: This won't need, can delete it

Comment (mfmsuhail) [inline variables.tf:56]: No need of variable reserve_nat_eip

File: main.tf
Comment (mfmsuhail) [inline main.tf:1]: what is the use of defining "count = 1"

Comment (aabidh-op) [reply main.tf:1]: can be removed

**Review by mfmsuhail — APPROVED** [review]

**Pull request merged**